app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  include DeviseTokenAuth::Concerns::User
+
+  validates :first_name, presence: true
+  validates :last_name, presence: true
```

Comment:
Keep in mind that if you are gonna validate the presence of these 2 fields in the model, then the DB should have the same validation